src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Lottery().getRandomBall();
+        Ball secondBall = new Lottery().getRandomBall();
+        Ball thirdBall = new Lottery().getRandomBall();
+
+        System.out.println(firstBall);
+        System.out.println(secondBall);
+        System.out.println(thirdBall);
```

Comment:
Lets use only one Lottery object with loop to generate and print objects of Ball here